examples/webpack/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "webpack",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "unplugin-vue-router": "workspace:*",
+    "vue": "^3.2.13",
+    "vue-router": "^4.1.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
```

Comment:
let's remove this dependency unless completely necessary. it's quite heavy

Reply:
As you suggested I have removed,  but removing this dependency will pop up a warning during installation.

<img width="460" alt="image" src="https://user-images.githubusercontent.com/31278178/178923458-573c2628-c037-4986-817e-de5fa4fdd668.png">

Reply:
Maybe it's worth basing it in a new project:

<img width="729" alt="Screenshot 2022-07-14 at 10 13 28" src="https://user-images.githubusercontent.com/664177/178935145-c29ea38f-7e71-4ef6-b719-60096f094d6d.png">